azkaban-common/src/main/java/azkaban/utils/PropsUtils.java

```diff
@@ -229,8 +233,8 @@ private static String resolveVariableExpression(final String value) {
   }
 
   /**
-   * Function that looks for expressions to parse. It parses backwards to
-   * capture embedded expressions
+   * Function that looks for expressions to parse. It parses backwards to capture embedded
+   * expressions
```

Comment:
No change, this is still how the javadoc lines are being cut. Leaving like this then.